org.inkscape.Inkscape.json

```diff
@@ -0,0 +1,194 @@
+        {
+            "name": "glibmm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/glibmm/2.48/glibmm-2.48.1.tar.xz",
+                    "sha256": "dc225f7d2f466479766332483ea78f82dc349d59399d30c00de50e5073157cdf"
+                }
+            ]
+        },
+        {
+            "name": "pangomm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/pangomm/2.40/pangomm-2.40.1.tar.xz",
+                    "sha256": "9762ee2a2d5781be6797448d4dd2383ce14907159b30bc12bf6b08e7227be3af"
+                }
+            ]
+        },
+        {
+            "name": "atkmm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/atkmm/2.24/atkmm-2.24.2.tar.xz",
+                    "sha256": "ff95385759e2af23828d4056356f25376cfabc41e690ac1df055371537e458bd"
+                }
+            ]
+        },
+        {
+            "name": "gtkmm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/gtkmm/2.24/gtkmm-2.24.5.tar.xz",
+                    "sha256": "0680a53b7bf90b4e4bf444d1d89e6df41c777e0bacc96e9c09fc4dd2f5fe6b72"
+                }
+            ]
+        },
+        {
+            "name": "potrace",
+            "config-opts": ["--with-libpotrace"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://potrace.sourceforge.net/download/1.14/potrace-1.14.tar.gz",
+                    "sha256": "db72b65311cfdcb63880b317f610d84485f086e15f88ca2346012d49414cd97e"
+                }
+            ]
+        },
+        {
+            "name": "lxml",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python ./setup.py build",
+                "python ./setup.py install --prefix=/app --optimize=1"
+            ],
+            "build-options": {
+                "env": {
+                    "XSLT_CONFIG": "pkg-config libxslt"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/39/e8/a8e0b1fa65dd021d48fe21464f71783655f39a41f218293c1c590d54eb82/lxml-3.7.3.tar.gz",
+                    "sha256": "aa502d78a51ee7d127b4824ff96500f0181d3c7826e6ee7b800d068be79361c7"
+                }
+            ]
+        },
+        {
+            "name": "inkscape",
+            "post-install": [
+                "mkdir -p /app/share/app-info/xmls",
+                "mv /app/share/appdata/inkscape.appdata.xml /app/share/app-info/xmls/org.inkscape.Inkscape.xml",
+                "gzip /app/share/app-info/xmls/org.inkscape.Inkscape.xml"
```

Comment:
Because the `build-update-repo` command does not work without this for some reason:
```
flatpak build-update-repo --generate-static-deltas --gpg-sign=DD8968B2F64F0260A3875EE0018A14794D3A8106 repo
Appstream-Zweig wird aktualisiert
No appstream data for app/org.inkscape.Inkscape/x86_64/master: No such file or directory: /files/share/app-info
```

Also, should I worried about this: 

```
(flatpak-builder:9424): flatpak-WARNING **: Unknown property rename-appdata for type BuilderManifest
```
?